Src/UnitTests/CoravelUnitTests/Scheduling/Invocable/InvocableTests.cs

```diff
@@ -63,6 +63,56 @@ public async Task TestScheduledInvocableWithMissingParamsDoesNotRun()
             Assert.False(invocableRan);
         }
 
+        [Fact]
+        public async Task TestScheduledInvocableWithParamsTypeRuns()
+        {
+            bool invocableRan = false;
+            var services = new ServiceCollection();
+            services.AddScoped<Action>(p => () => invocableRan = true);
+            services.AddScoped<TestInvocableWithParams>();
+            var provider = services.BuildServiceProvider();
+
+            var scheduler = new Scheduler(new InMemoryMutex(), provider.GetRequiredService<IServiceScopeFactory>(), new DispatcherStub());
+            scheduler.ScheduleWithParams(typeof(TestInvocableWithParams), "stringParam", 1).EveryMinute();
+
+            await scheduler.RunAtAsync(new DateTime (2019, 1, 1));
+
+            Assert.True (invocableRan);
+        }
+
+        [Fact]
+        public async Task TestScheduledInvocableWithMissingParamsTypeDoesNotRun()
+        {
+            bool invocableRan = false;
+            var services = new ServiceCollection();
+            services.AddScoped<Action>(p => () => invocableRan = true);
+            services.AddScoped<TestInvocableWithParams>();
+            var provider = services.BuildServiceProvider();
+
+            var scheduler = new Scheduler(new InMemoryMutex(), provider.GetRequiredService<IServiceScopeFactory>(), new DispatcherStub());
+            scheduler.ScheduleWithParams(typeof(TestInvocableWithParams), "stringParam").EveryMinute();
+            
+            await scheduler.RunAtAsync(new DateTime (2019, 1, 1));
+
+            Assert.False (invocableRan);
+        }
+
+        [Fact]
+        public async Task TestScheduledInvocableWithParamsType_Throws()
+        {
+            await Assert.ThrowsAsync<ArgumentException>("invocableType", async () =>
```

Comment:
Why don't we wrap the specific line that will throw. This would make it much more explicit what line(s) is expected to throw.  Also, it would avoid any false positives in the future (i.e. if another part of this block threw a similar exception, but wasn't from the specific method we were trying to test.)

Thanks!

Reply:
Addressed.